### PR TITLE
Bintray2

### PR DESCRIPTION
**Pull request by MasseGuillaume**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,17 +15,20 @@ In `project/plugins.sbt`:
...
-addSbtPlugin("org.scala-native" % "sbt-cross"         % "0.1.0-SNAPSHOT") // (2)
-addSbtPlugin("org.scala-native" % "sbt-scalajs-cross" % "0.1.0-SNAPSHOT") // (3)
-addSbtPlugin("org.scala-native" % "sbt-scala-native"  % "0.1.0-SNAPSHOT") // (4)
+resolvers += Resolver.url(
+        "bintray-scala-native-sbt-plugins",
+        url("http://dl.bintray.com/scala-native/sbt-plugins"))(
+        Resolver.ivyStylePatterns)                                               // (1)
+addSbtPlugin("org.scala-native" % "sbt-crossproject"         % "0.1.0-SNAPSHOT") // (2)
+addSbtPlugin("org.scala-native" % "sbt-scalajs-crossproject" % "0.1.0-SNAPSHOT") // (3)
+addSbtPlugin("org.scala-native" % "sbt-scala-native"         % "0.1.0-SNAPSHOT") // (4)
```
Comment (densh) [inline README.md]: `0.1.0`, without a snapshot.

File: sbt-crossproject-test/src/sbt-test/plugins.sbt [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  "bintray-scala-native-sbt-plugins",
+  url("http://dl.bintray.com/scala-native/sbt-plugins"))(
+  Resolver.ivyStylePatterns)
+
+val pluginVersion = sys.props.get("plugin.version").getOrElse("0.1.0-SNAPSHOT")
+
+addSbtPlugin("org.scala-js"     % "sbt-scalajs"              % "0.6.13")
+addSbtPlugin("org.scala-native" % "sbt-scalajs-crossproject" % pluginVersion)
+addSbtPlugin("org.scala-native" % "sbt-crossproject"         % pluginVersion)
+addSbtPlugin("org.scala-native" % "sbt-scala-native"         % "0.0.0.1")
```
Comment (densh) [inline sbt-crossproject-test/src/sbt-test/plugins.sbt]: `0.1.0`

**Comment by MasseGuillaume** [comment]
@densh I'm on it ;-). I'm a bit stuck with caching issue running the test. I should be done soon ;-).

**Comment by MasseGuillaume** [comment]
Great this one passes!

I suggest to tag this 0.1.0 and re-release on Bintray.

I bumped Scala.js from 0.6.13 to 0.6.14

**Comment by MasseGuillaume** [comment]
just removed the resolver :)

**Comment by MasseGuillaume** [comment]
It look like removing the resolver does not work for sbt-scala-native. I sent an email to Bintray about this.

* `curl -vvv https://dl.bintray.com/scala-native/sbt-plugins/org.scala-native/sbt-scala-native/scala_2.10/sbt_0.13/0.1.0/ivys/ivy.xml` our repository works
* `curl -vvv https://dl.bintray.com/sbt/sbt-plugin-releases/org.scala-native/sbt-scala-native/scala_2.10/sbt_0.13/0.1.0/ivys/ivy.xml` sbt/sbt-plugins-releases does not work
* `curl -vvv https://dl.bintray.com/sbt/sbt-plugin-releases/org.scala-native/sbt-crossproject/scala_2.10/sbt_0.13/0.1.0/ivys/ivy.xml` sbt/sbt-plugins-releases works for sbt-crossproject

**Comment by densh** [comment]
@MasseGuillaume Is there a still a problem here? I was able to resolve all the things without extra resolvers  for my demo game build after nuking ivy cache completely.

**Comment by MasseGuillaume** [comment]
@densh restarting the build. The issue above should be fixed

**Comment by MasseGuillaume** [comment]
@densh Great the ci pass on linux!

**Comment by densh** [comment]
Fantastic work, LGTM!

**Pull request merged**